### ISSUE TITLE
Problem with metrics=["Accuracy"] in compile

### ISSUE THREAD

**Issue by mmoradik** [fchollet/keras#3096]
Hi, I am writing a code for multi-label classification using LSTM. 
model.compile (loss='categorical_crossentropy', optimizer='rmsprop',class_mode='categorical' ,metrics=["Accuracy"]). 

The error that I have received is:
compile() got an unexpected keyword argument 'metrics' a comment

Any idea!!!

**Comment by mmoradik** [comment]
Hi, I am writing a code for multi-label classification using LSTM. 
model.compile(loss='categorical_crossentropy', optimizer='rmsprop',class_mode='categorical' ,metrics=["Accuracy"]). 

The error That I have received is:
compile() got an unexpected keyword argument 'metrics' a comment

**Comment by fchollet** [comment]
Just update Keras.

**Comment by madhumita-git** [comment]
I have the same bug, and I am using Python 3.4. I just did a fresh install of keras through the github repo, but it does not solve the problem.

**Comment by madhumita-git** [comment]
There was a version conflict due to upgrade. Removing the older versions from my distribution fixed the problem!

**Comment by mayaghaei** [comment]
Can I ask you to provide more information how you resolved the version conflict problem?

**Comment by madhumita-git** [comment]
@mayaghaei I deleted the old version from the local path of python packages
